maskrcnn_benchmark/modeling/detector/generalized_rcnn.py

```diff
@@ -61,5 +64,7 @@ def forward(self, images, targets=None):
             losses.update(detector_losses)
             losses.update(proposal_losses)
             return losses
-
-        return result
+        if return_features:
```

Comment:
Instead of hard-coding the last features, what do you think about the user attaching a hook in their model to obtain the output of a particular layer? This would be a non-intrusive change to this file, and the feature that the user could obtain can be arbitrary

Reply:
That would be much more logical. But I couldn't come up with a better approach at the time. Adding [hook](https://pytorch.org/docs/stable/nn.html#torch.nn.Module.register_forward_hook) didn't come to my mind. I'll amend the PR with this method.